src/main/kotlin/Main.kt

```diff
@@ -1,5 +1,5 @@
 fun main() {
-    println("Enter your city…")
+    println("Enter your city...")
     val cityName = readLine()
     println("User lives in $cityName")
 }
```

Comment:
keep 1 extra line at the EOF
```suggestion
}

```